File: Backjun/bronze5/b3003/b3003.py
# 킹, 퀸, 룩, 비숍, 나이트, 폰
# https://www.acmicpc.net/problem/3003

pieces_of_right = [1, 1, 2, 2, 2, 8]

pieces_of_input = list(map(int, input().split()))

results = []

for i in range(6):
    results.append(pieces_of_right[i] - pieces_of_input[i])

print(" ".join(map(str, results)))

File: Backjun/bronze5/b3003/b3003.c
// 킹, 퀸, 룩, 비숍, 나이트, 폰
// https://www.acmicpc.net/problem/3003

#include <stdio.h>

int main() {
    int piecesOfInput[6];  // king, queen, rook, bishop, knight, pawn
    int piecesOfRight[6] = {1, 1, 2, 2, 2, 8};
    int results[6];

    for(int i = 0; i < 6; i++) {
        scanf("%d", &piecesOfInput[i]);
    }

    for (int i = 0; i<6; i++) {
        results[i] = piecesOfRight[i] - piecesOfInput[i];
        printf("%d ", results[i]);
    }
    printf("\n");

    return 0;
}